Sources/LocalizationTools/main.swift

```diff
@@ -38,16 +41,27 @@ struct LocalizationTools: ParsableCommand {
     mutating func run() throws {
         guard validateArguments() else { Self.exit() }
         
-        let shippingLocales = URL(fileURLWithPath: projectPath).deletingLastPathComponent().appendingPathComponent("shipping_locales.txt")
-        let locales = try! String(contentsOf: shippingLocales).components(separatedBy: .newlines).filter { !$0.isEmpty }
-
+        var locales: [String]
 
+        if localeCode != nil {
+            locales = [ localeCode! ]
+        } else {
+            let directoryContent = try FileManager.default.contentsOfDirectory(at: URL(fileURLWithPath: l10nProjectPath), includingPropertiesForKeys: nil, options: .skipsHiddenFiles)
```

Comment:
I don't know swift, but this seems incredibly clunky. Hopefully an actual developer has a better idea on how to get the list of non hidden folders in l10nProjectPath :-(

Reply:
I haven't worked much with the FileManager, but this seems correct, in general.